packages/grafana-ui/src/components/AutoSizeInput/AutoSizeInput.story.tsx

```diff
@@ -0,0 +1,101 @@
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+
+import { KeyValue } from '@grafana/data';
+import { Icon, Button, AutoSizeInput } from '@grafana/ui';
+
+import { getAvailableIcons, IconName } from '../../types';
+import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
+
+import mdx from './AutoSizeInput.mdx';
+
+const prefixSuffixOpts = {
+  None: null,
+  Text: '$',
+  ...getAvailableIcons().reduce<KeyValue<string>>((prev, c) => {
```

Comment:
We have something hidden away that looks (almost) the same as required here. Maybe it's possible to reuse it? https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/utils/storybook/knobs.ts#L7-L15